smoke-test/pom.xml

```diff
@@ -377,6 +377,12 @@
       <version>3.0.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Lombok needs to present only during compile time, see als: https://projectlombok.org/setup/maven